ext/nnstreamer/tensor_filter/tensor_filter_snap.cc

```diff
@@ -0,0 +1,1043 @@
+/**
+ * SPDX-License-Identifier: LGPL-2.1-only
```

Comment:
Incorrect SPDX identifier. Add it to the top line.

Reply:
updated